src/wallet/rpcwallet.cpp

```diff
@@ -38,6 +38,7 @@
 
 
 static const std::string WALLET_ENDPOINT_BASE = "/wallet/";
+static const std::string HELP_REQUIRING_PASSPHRASE{"\nRequires wallet passphrase to be set with walletpassphrase call if wallet is encrypted.\n"};
```

Comment:
nit: The line above is not using a C++11 list init. I would change it to a C one to keep it coherent.